tests/tensorflow/targets.jsonnet

```diff
@@ -22,9 +22,5 @@ local r2_9 = import 'r2.9/targets.jsonnet';
 // Add new versions here
 std.flattenArrays([
   nightly,
-  r2_7,
```

Comment:
Please revert this file -- #743 already removes these tests.

Reply:
I think you just changed the schedule. While I was trying not to push these workloads to GKE. Let me know if that makes sense.